pypeit/reduce.py

```diff
@@ -239,7 +239,7 @@ def find_objects_pypeline(self, image, ivar, std=False, std_trace=None, maskslit
          """
         return None, None, None
 
-    def global_skysub(self, sciimg, sciivar, tilts, std=False, skymask=None, update_crmask=True, maskslits=None, show_fit=False,
+    def global_skysub(self, tilts, std=False, skymask=None, update_crmask=True, maskslits=None, show_fit=False,
```

Comment:
Please restore the original argument lists.

Reply:
Done for `find_obj`